data/json/items/gun/45.json

```
@@ -528,5 +528,53 @@
       { "magazine_well": "107 ml", "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "glock_21mag", "glock_21mag26" ] }
     ],
     "melee_damage": { "bash": 8 }
+  },
+  {
+    "id": "greasegun",
+    "looks_like": "hk_mp5",
+    "//": "When or if all guns will have barrel lenght as damage modifier factor, barrel lenght is 203mm",
```

```suggestion
    "barrel_length": "203 mm"
```
Shouldn't you just futureproof this instead of needing to go over it again after #65756

---

Well, good point. But "when the future comes" someone still need to remove "ranged_damage": { "damage_type": "bullet", "amount": X } " modifier.
Fixed